docs/user-management.md

```diff
@@ -0,0 +1,146 @@
+
+Navigate to `/admin/admin_users` to manage accounts:
+
+- **Create**: Click *New admin user*, supply name and email, and submit. Koi creates the record without a password; use *Generate login link* to send them an activation URL.
+- **Edit**: Admins can update each other’s names/emails. When editing yourself (`request.variant = :self`), the form also exposes a password field so you can set or change a password.
+- **Archive / Restore / Delete**: First archive an admin to revoke access (sets `archived_at`). Archived admins are hidden from default queries; use the *Archived* tab to restore or permanently delete them (hard delete only works from the archived list).
+- **Audit**: The show view surfaces `last_sign_in_at`, passkey status, MFA enrolment, and whether the account is archived.
+
+**Verify**
+
+```sh
+bin/rails runner 'admin = Admin::User.order(:created_at).first; puts({ email: admin.email, archived: admin.archived?, passkeys: admin.credentials.count })'
+```
+
+Expect `archived` to be `false` for active users and `passkeys` to reflect any registered credentials.
+
+## Managing Passkeys and MFA
+
+From an admin’s profile (accessible via the header avatar or `/admin/admin_users/:id`):
+
+- Choose *New passkey* to open the WebAuthn registration dialog. The browser prompts for a device, and Koi stores the resulting credential in `admin_credentials`. You can name each credential to track which device it belongs to and remove stale ones individually.
+- Choose *Add* under MFA to start enrolment. Koi pre-generates an `otp_secret`, renders a QR code (via `RQRCode`), and asks for a verification token. Once saved, future password logins require the 6-digit code. Removing MFA deletes the secret immediately.
+
+Passkeys are the recommended option when available; MFA is a good fallback for browsers or devices that cannot store passkeys.
+
+## Inviting Colleagues and Handling Approvals
+
+There is no separate approval workflow. To onboard someone:
+
+1. Create the admin via the CLI or `/admin/admin_users/new`.
+2. On the admin’s show page, click *Generate login link*. This hits `Admin::TokensController#create` and renders a copyable `/admin/session/token/<token>` URL.
+3. Share the link over a secure channel. The recipient visits the link, reviews their details, and clicks *Sign in* to consume the token.
+4. Encourage them to add a passkey (preferred) or set a password and MFA once signed in.
+
+Tokens expire 30 minutes after generation or immediately after use. If a token expires before it is consumed, issue a new one from the same page or by running `bin/admin-reset`.
+
+## Automatic Login and Redirects in Development
+
+With auto-login enabled, Koi tries to sign you in using the seeded email. If the admin record no longer matches, you will be redirected to `/admin/session/new` repeatedly.
+
+### Avoiding the Redirect Loop
+
+- **Add a matching account first**:
+  ```sh
+  bin/admin-adduser -n "Your Name" -e "#{ENV['USER']}@katalyst.com.au"
```

Comment:
did you run `bin/setup` after checking out Koi? There's a `db/seeds.rb` that creates this user for you automatically.